dolphinscheduler-python/pydolphinscheduler/docs/source/tasks/sagemaker.rst

```diff
@@ -0,0 +1,34 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+SageMaker
+=========
+
+
+A shell task type's example and dive into information of **PyDolphinScheduler**.
```

Comment:
It is seem you do not change the task type here, could you add one more PR to correct it? @jieguangzhou

Reply:
it should be `A SageMaker task types`

Reply:
Ok, I will revise it as soon as possible

Reply:
fixed by: https://github.com/apache/dolphinscheduler/pull/11109